src/browser.ts

```diff
@@ -101,6 +132,20 @@ export class GrpcClient {
     );
   }
 
+  /**
+   * Browser version of createStub
+   * Creates a gRPC-fallback stub and auth
+   * @param {function} CreateStub - The constructor function of the stub.
+   * @param {Object} service - [TODO]
+   * @param {Object} options - The optional arguments to customize
+   *   gRPC connection. This options will be passed to the constructor of
+   *   gRPC client too.
+   * @param {string} options.servicePath - The name of the server of the service.
```

Comment:
Let's better call it `The hostname of the API endpoint service`. The word 'server' is really overused and can mean tons of different things :)